pkg/common/sign/sign.go

```diff
@@ -0,0 +1,108 @@
+package sign
```

Comment:
some thoughts/nits about naming:
- sign.Singer seems repetitive, maybe `cert.Signer` would work better?
- client doesn't use this package, only the server, so maybe it should live in the server namespace?